src/helpers/txHelper.ts

```diff
@@ -15,6 +15,66 @@ import { LedgerErrorResponse } from "@/types/types";
 
 import { validate } from "@/validate";
 
+/* Build Transaction related */
+export type FeeBumpedTxResponse = {
+  errors: string[];
+  xdr: string;
+};
+
+const buildFeeBumpTx = ({
+  innerTxXDR,
```

Comment:
In other places, we use camelcase for `XDR` -> `Xdr`.

```suggestion
  innerTxXdr,
```